agents/pve/fence_pve.py

```diff
@@ -54,6 +54,13 @@ def set_power_status(conn, options):
 	send_cmd(options, cmd, post={"skiplock":1})
 
 
+def reboot_cycle(conn, options):
+	del conn
+	cmd = "nodes/" + options["--nodename"] + "/qemu/" + options["--plug"] + "/status/reset"
```

Comment:
You should use `options["--vmtype"]` like in `set_power_status()` instead of `/qemu/`, so it can be used for all supported vmtypes.

Reply:
It's a fair point, and I can certainly make the change. It won't make much difference, though, since the reset action is only available for the qemu machine type anyway.

Ideally, fence_pve should notice the mismatch and gracefully fall back to the onoff method for lxc vmtypes.

Reply:
Oh. I can add some code to do that.